models/attachment.go

```diff
@@ -221,20 +222,20 @@ func DeleteAttachmentsByComment(commentID int64, remove bool) (int, error) {
 		return 0, err
 	}
 
-	return DeleteAttachments(db.DefaultContext(), attachments, remove)
+	return DeleteAttachments(db.DefaultContext, attachments, remove)
 }
 
 // UpdateAttachment updates the given attachment in database
 func UpdateAttachment(atta *Attachment) error {
-	return updateAttachment(db.DefaultContext().Engine(), atta)
+	return updateAttachment(db.GetEngine(db.DefaultContext), atta)
 }
 
 // UpdateAttachmentByUUID Updates attachment via uuid
-func UpdateAttachmentByUUID(ctx *db.Context, attach *Attachment, cols ...string) error {
+func UpdateAttachmentByUUID(ctx context.Context, attach *Attachment, cols ...string) error {
 	if attach.UUID == "" {
-		return fmt.Errorf("Attachement uuid should not blank")
+		return fmt.Errorf("attachment uuid should not blank")
```

Comment:
Should that be lowercase? Could add a `be`.

Reply:
It should be lowercase - golang errors are not supposed to start with a capital letter: https://github.com/golang/go/wiki/CodeReviewComments#error-string

Although I noted that the verb was missing I'd have to double check that nothing is actually checking for the string as is before adding in the `be`